gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "13.1.8",
+  "version": "13.2.0",
```

Comment:
Since React is so core of a library, I consider upgrading it to be more than a patch change.